common/cosmwasm-smart-contracts/mixnet-contract/src/mixnode.rs

```diff
@@ -247,24 +247,24 @@ impl DelegatorRewardParams {
     }
 }
 
-#[derive(Debug, Clone, JsonSchema, PartialEq, Serialize, Deserialize, Copy)]
+#[derive(Debug, Clone, JsonSchema, PartialEq, Serialize, Deserialize)]
 pub struct StoredNodeRewardResult {
     reward: Uint128,
-    lambda: Uint128,
-    sigma: Uint128,
+    lambda: String,
```

Comment:
just leaving a comment here to remember to [presumably] revert it back to Uint128 before merging

Reply:
No, no, leave it as string, Uint128 will truncate the float

Reply:
Never mind, I see what you did there, will test it out in QA